### PR TITLE
[Ubuntu] Update ansible-base to ansible-core

### PR DESCRIPTION
**Pull request by al-cheb**
# Description
Update `ansible-base` 2.10(only exists for version 2.10 and in Ansible 3) to `ansible-core` 2.11[(roadmap](https://docs.ansible.com/ansible-core/devel/roadmap/ROADMAP_2_11.html), [changelog](https://github.com/ansible/ansible/blob/stable-2.11/changelogs/CHANGELOG-v2.11.rst#release-summary)).

#### Related issue:
https://github.com/actions/virtual-environments/issues/3708

## Check list
- [x] Related issue / work item is attached
- [x] Tests are written (if applicable)
- [x] Documentation is updated (if applicable)
- [x] Changes are tested and related VM images are successfully generated

### CODE REVIEW

File: images/linux/scripts/installers/pipx-packages.sh [hunk 1]
```diff
@@ -19,9 +19,10 @@ for package in $pipx_packages; do
         echo "Install $package into default python"
         pipx install $package
 
-        # Install ansible into an existing ansible-base Virtual Environment
-        if [[ $package == "ansible-base" ]]; then
-            pipx inject ansible-base ansible
+        # https://docs.ansible.com/ansible/latest/installation_guide/intro_installation.html
+        # Install ansible into an existing ansible-core Virtual Environment
+        if [[ $package == "ansible-core" ]]; then
+            pipx inject ansible-core ansible
```
Comment (miketimofeev) [inline images/linux/scripts/installers/pipx-packages.sh:25]: Maybe use the variable?
```suggestion
            pipx inject $package ansible
```

Comment (al-cheb) [reply images/linux/scripts/installers/pipx-packages.sh]: Thx. Fixed.

**Review by AlenaSviridenko — DISMISSED** [review]

**Review by MaksimZhukov — APPROVED** [review]

File: images/linux/scripts/SoftwareReport/SoftwareReport.Tools.psm1 [hunk 2]
```diff
@@ -1,5 +1,5 @@
 function Get-AnsibleVersion {
-    $ansibleVersion = ansible --version | Select-Object -First 1 | Take-OutputPart -Part 1
+    $ansibleVersion = (ansible --version)[0] -replace "[^\d.]"
     return "Ansible $ansibleVersion"
```
Comment (miketimofeev) [inline images/linux/scripts/SoftwareReport/SoftwareReport.Tools.psm1:3]: What do you think does it make sense to rename it to Ansible-core for visibility?

Comment (al-cheb) [reply images/linux/scripts/SoftwareReport/SoftwareReport.Tools.psm1:3]: I prefer just Ansible without core. Maybe it makes sense to add src, like that Ansible x.x.x (pipx: ansible-core). Let's vote: @maxim-lobanov , @AlenaSviridenko, @MaksimZhukov

Comment (maxim-lobanov) [reply images/linux/scripts/SoftwareReport/SoftwareReport.Tools.psm1:3]: I prefer Ansible without core too.

**Review by miketimofeev — APPROVED** [review]

File: images/linux/scripts/SoftwareReport/SoftwareReport.Tools.psm1 (hunk 2 above)
Comment (AlenaSviridenko) [reply images/linux/scripts/SoftwareReport/SoftwareReport.Tools.psm1:3]: But it is not ansible itself, isn't it? it is the ansible-core version

**Pull request merged**